src/kimmdy/cmd.py

```diff
@@ -291,13 +268,14 @@ def _run(args: argparse.Namespace):
             raise e
     finally:
         logging.shutdown()
+        if args.generate_jobscript:
+            shutil.rmtree(config.out)
```

Comment:
are you sure we want to delete the output directory without question when a jobscript is created? 
One might also want to generate a jobscript for a new run, keeping the existing output and just incrementing the number or one might want to create a jobscript to continue and existing run that started locally, which we then definitely don't want to remove.

Reply:
I thought kimmdy always creates a new run directory when `kimmdy.cmd._run` is called.`config.out` is then set to that directory name. When a jobscript is generated, we get a new run directory but rmgr.run is never called because of the `if ... else`. This means we always get an almost empty run directory with only some logging inside when using `--generate-jobscript`. 
Using `jobscript.sh` to continue a run would create a new run directory and not end with removing that directory with this logic.